test/python/visualization/test_pass_manager_drawer.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests for pass manager visualization tool."""
+
+import unittest
+import os
+
+from qiskit.transpiler.transpile_config import TranspileConfig
+from qiskit.transpiler import CouplingMap, Layout
+from qiskit import QuantumRegister
+from qiskit.transpiler.preset_passmanagers import level_0_pass_manager, level_1_pass_manager
+from qiskit.transpiler.passes import SetLayout, CheckMap, EnlargeWithAncilla, RemoveResetInZeroState
+
+from qiskit.visualization.pass_manager_visualization import HAS_GRAPHVIZ
```

Comment:
I think it might be better to do it the way I have done it, as this way both the testing and the actual code can use the same check. This way if it ever needs to be checked in the future it only has to be done in one place. It mirrors how we check for matplotlib in [that circuit drawer](https://github.com/Qiskit/qiskit-terra/blob/6d6fa1c3eb40c0a5de60123b0f90e5241c7d282d/qiskit/visualization/matplotlib.py#L28).